site/components/form/CsrfForm.tsx

```diff
@@ -9,8 +9,8 @@ interface CsrfFormProps {
 }
```

Comment:
Could we make csrfToken string or undefined, and default it in the component to an empty string? So that we dont need to always do csrfToken={csrfToken || ""}